tox.ini

```diff
@@ -1041,3 +1042,46 @@ commands =
     tox -e dev.k8s.destroy
     bash -c 'k3d registry delete registry.localhost || true'
     sudo rm -rf ~/.k3d-registry
+
+[testenv:backend.test.basecpu]
+description = Base CPU Docker Image Test
+changedir = {toxinidir}/packages
+allowlist_externals =
+    docker
+    bash
+commands =
+    bash -c 'docker rm $(docker ps -aq) --force || true'
+    bash -c 'docker volume prune -f || true'
+    bash -c 'docker buildx use default || true'
+    ; Build the base image
+    bash -c 'docker build -f ./grid/backend/syft_base_cpu.dockerfile . -t cpu-worker:latest'
+    bash -c 'docker rmi cpu-worker:latest'
+
+    bash -c 'docker rm $(docker ps -aq) --force || true'
+    bash -c 'docker volume prune -f || true'
+    bash -c 'docker buildx use default || true'
+    bash -c 'docker build -f grid/backend/syft_base_cpu.dockerfile . -t cpu-worker:llama  --build-arg PIP_PACKAGES="llama-index" --build-arg SYSTEM_PACKAGES="htop"'
+    bash -c 'docker run --rm cpu-worker:llama pip list | grep llama-index'
+    bash -c 'docker run --rm cpu-worker:llama apk -e info htop'
+    bash -c 'docker rmi cpu-worker:llama'
+
+    bash -c 'docker rm $(docker ps -aq) --force || true'
+    bash -c 'docker volume prune -f || true'
+    bash -c 'docker buildx use default || true'
+    bash -c 'docker build -f grid/backend/syft_base_cpu.dockerfile . -t cpu-worker:opendp  --build-arg PIP_PACKAGES="llama-index opendp" --build-arg SYSTEM_PACKAGES="curl wget htop"'
+    bash -c 'docker run --rm cpu-worker:opendp pip list | grep llama-index'
+    bash -c 'docker run --rm cpu-worker:opendp pip list | grep opendp'
+    bash -c 'docker run --rm cpu-worker:opendp apk -e info htop'
+    bash -c 'docker run --rm cpu-worker:opendp apk -e info curl'
+    bash -c 'docker run --rm cpu-worker:opendp apk -e info wget'
+    bash -c 'docker rmi cpu-worker:opendp'
```

Comment:
Are these duplicated? Do we need L1060-L1066?

Reply:
Yep, we can only keep the latter (from line 1068).

Reply:
alr solved.